### ISSUE TITLE
为面向更多用户，期望开发相应的chrome拓展插件

### ISSUE THREAD

**Issue by zyrzjyzxy** [opendatalab/MinerU#226]
感谢项目开发者，这能应该是薄纱市面大多数同类剪藏工具和转.md的项目。
但是本项目主要面向的还是有linux基础的程序员。
为了面向更多非专业人士，我期望本项目未来能有**基于chrome的拓展插件（类似市面上的“简悦”），用于剪藏网页**。
由于我是小白，对该需求的可行性没有概念，但真诚希望本项目将来有相应方向的发展。

Here is Google Translate:
Thanks to the project developers, this should be the most similar cropping tool and .md project on the flash sale market.
However, this project is mainly aimed at programmers with basic knowledge of Linux.
In order to target more non-professionals, I hope that this project will have a chrome-based extension in the future (similar to &quot;Jian Yue&quot; on the market) for cutting web pages.
Since I am a newbie, I am not sure about the feasibility of this requirement, but I sincerely hope that this project can develop in the corresponding direction in the future.

**Comment by Kain-90** [comment]
Good idea, would like to discuss, now online pdf to markdown has a lot, I haven't thought about what the main benefit of a browser extension using that feature would be.

**Comment by zyrzjyzxy** [comment]
感谢您的反馈😇。
根据我的生活体验和工作需求而言，我能想到至少以下两点：
1.PDF的灵活性较差。若想要在PDF文档上做笔记或其他用途，在市面上的主流软件的基础上，比较麻烦。而Markdown正是以其便捷性而著称，若将pdf转到markdown，则能十分便利地记录并修改PDF上的信息。
2.浏览器扩展便于操作。若能直接在浏览器上，通过右键，就能把PDF文档（论文、试卷等）保存到本地或线上，而不是像本项目一样命令行操作，将大大减低操作门槛和缩短操作时间。

Here is Google Translate:

Thanks for your feedback😇.
Based on my life experience and work needs, I can think of at least the following two points:
1.PDF is less flexible. If you want to take notes on PDF documents or for other purposes, it is more troublesome based on the mainstream software on the market. Markdown is famous for its convenience. If you convert PDF to markdown, you can easily record and modify the information on the PDF.
2. Browser extension facilitates operation. If you can save PDF documents (papers, test papers, etc.) locally or online directly on the browser by right-clicking, instead of command line operations like this project, it will greatly reduce the operation threshold and shorten the operation time.

**Comment by Kain-90** [comment]
Thanks for the additional clarification, and in response to the first point, I think I can see how converting to markdown would make it easier to take notes, edit, and record the final result on mainstream note-taking software, right?

The second point is still a little incomprehensible, pdf downloads can be done without browser extensions, right?

**Comment by JustDoIt166** [comment]
> 感谢项目开发者，这能应该是薄纱市面大多数同类剪藏工具和转.md的项目。
> 但是本项目主要面向的还是有linux基础的程序员。
> 为了面向更多非专业人士，我期望本项目未来能有**基于chrome的拓展插件（类似市面上的“简悦”），用于剪藏网页**。
> 由于我是小白，对该需求的可行性没有概念，但真诚希望本项目将来有相应方向的发展。
> 
> Here is Google Translate:
> Thanks to the project developers, this should be the most similar cropping tool and .md project on the flash sale market.
> However, this project is mainly aimed at programmers with basic knowledge of Linux.
> In order to target more non-professionals, I hope that this project will have a chrome-based extension in the future (similar to &quot;Jian Yue&quot; on the market) for cutting web pages.
> Since I am a newbie, I am not sure about the feasibility of this requirement, but I sincerely hope that this project can develop in the corresponding direction in the future.

你提供服务器跑模型倒是可以直接上传识别

**Comment by myhloli** [comment]
目前我们提供全平台的客户端方便用户使用，可以在官网 https://mineru.net/ 直接下载使用。
